src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -155,14 +154,13 @@ class OrganizationIntegrations extends AsyncComponent {
     const {appInstalls} = this.state;
 
     return (
-      <IntegrationRow key={`row-${key}`}>
-        <SentryAppInstallations
-          key={key}
```

Comment:
I checked `IntegrationRow` as well as `ProviderRow` to make sure we aren't using `key`. everything looks good.

Reply:
`key` can't be used at all outside of React internals (and enzyme, as the old tests were doing).